core/coreunsafe/reset.go

```diff
@@ -74,6 +75,16 @@ func ResetEverything(ctx context.Context, db pg.DB, rDB *raft.Service) error {
 		return errors.Wrap(err)
 	}
 
+	// Delete all grants in raft storage
+	for _, p := range core.Policies {
+		err = rDB.Delete(ctx, core.GrantPrefix+p)
+		if err != nil {
+			return errors.Wrapf(err, "could not delete grants for policy %s from RaftDB", p)
+		}
+	}
+
+	// TODO(tessr): remove this core from the allowed members list in raft, too, potentially
```

Comment:
Maybe we should just delete the whole allowed members list? If reset is only available on single-node clusters it prob doesn't matter but maybe we should delete it just to be consistent?